### DISCUSSION TITLE
How the conversion tracking  embeds like Facebook Pixel, LinkedIn Insights tag, Twitter pixel, gtag js and google analytics will change after phasing out of 3rd party cookies?

### DISCUSSION THREAD

**Discussion by krishna-kudari** [privacysandbox/privacy-sandbox-dev-support#442] [Phasing out third-party cookies]
We are using Facebook Pixel, LinkedIn Insights tag, Twitter pixel, gtag js and google analytics embeds in our website for tracking conversions from the ad campaigns.

with the phase out of 3rd party cookies, will the above scripts stop functioning?
as a developer what are the steps I should take?
does these scripts able to track conversions in case of incognito mode?

I have tried disabling the 3rd party cookies in the chrome settings and still I am seeing the events/ api calls being fired in network panel of dev tools and resulting in 200 status code, but i can see a tooltip saying "The attempt to set cookie was blocked due to user preferences" but still I am able to see cookies in the cookie panel. how this is possible?
<img width="819" alt="image" src="https://github.com/user-attachments/assets/6018e9a9-e271-49c5-8819-9d993b450b33">

any insights will be extremely helpful

**Comment by shuranhuang** [comment]
Hi KRISHNA, blocking third-party cookies won't necessarily break a site's functionality. For the ads related stuff, it just means less effective advertising use cases (unless they've adopted the PS APIs already). In case it's helpful - [this page](https://github.com/privacysandbox/privacy-sandbox-dev-support/blob/main/3pc-migration-readiness.md) provides info of how sites are preparing for non-ad related cases. Though the coverage is limited right now, we expect more sites to contribute to this list. We are also working on presenting the readiness status of sites in an upcoming new DevTools panel.

Regarding your last question, as shown in the screenshot, the cookies were indeed blocked as the tooltip indicated, they are shown in the Network > cookie table just to provide more cookie access status information for debugging.
